static/elements/chromedash-banner.js

```diff
@@ -0,0 +1,59 @@
+import {LitElement, css, html} from 'lit-element';
+import {nothing} from 'lit-html';
+import SHARED_STYLES from '../css/shared.css';
+
+
+class ChromedashBanner extends LitElement {
+  static get properties() {
+    return {
+      timestamp: {type: Number},
```

Comment:
Tiny nit: maybe `timeInSeconds` for better readability.

Reply:
I think of "timestamp" is being more precise.  Timestamp is a unix timestamp, whereas "timeInSeconds" raises the question of "is that a unix timestamp or a some other measure of time?".   I added a comment to clarify that it is a unix timestamp.